docs/src/man/hosting.md

```diff
@@ -404,16 +409,17 @@ jobs:
           ref: gh-pages
       - name: Delete preview and history + push changes
         run: |
-            if [ -d "previews/PR$PRNUM" ]; then
+          if [ -d "${preview_dir}" ]; then
               git config user.name "Documenter.jl"
               git config user.email "documenter@juliadocs.github.io"
-              git rm -rf "previews/PR$PRNUM"
-              git commit -m "delete preview"
+              git rm -rf "${preview_dir}"
+              git commit -m "Delete preview for #${PR}"
               git branch gh-pages-new $(echo "delete history" | git commit-tree HEAD^{tree})
               git push --force origin gh-pages-new:gh-pages
```

Comment:
Could be problematic if a new PR pushes it's documentation while this workflow is running.